src/main/rx/expresslrs_common.h

```diff
@@ -54,8 +54,9 @@
 #define FREQ_HZ_TO_REG_VAL_900(freq) ((uint32_t)(freq / SX127x_FREQ_STEP))
 #define FREQ_HZ_TO_REG_VAL_24(freq) ((uint32_t)(freq / SX1280_FREQ_STEP))
 
-#define ELRS_RATE_MAX 4
-#define ELRS_BINDING_RATE_24 3
+#define ELRS_RATE_MAX_24 6
+#define ELRS_RATE_MAX_900 4
+#define ELRS_BINDING_RATE_24 5
```

Comment:
This one I could not figure out. If I used MAX (and included maths.h) I get this error:
```c
./src/main/common/maths.h:55:17: error: braced-group within expression allowed only inside a function
   55 |   __extension__ ({ __typeof__ (a) _a = (a); \
      |                 ^
```
I have no idea how to resolve this for use in a header's array declaration so I just rolled my own max. You're welcome to change it and school me on how to do it though.

Reply:
MAX is implemented more like inline function (each parameter is evaluated only once and return type is correctly derived. One disadvantage is that it can;t be used in every context. Your solution is fine. (another option is to add 'dumb' version of MAX macro, but naming it would be problematic)